### PR TITLE
WT-9217 Generate call log for conn.query_timestamp

### PR DESCRIPTION
**Pull request by Siddhartha8899**

### CODE REVIEW

**Review by bvpvamsikrishna — APPROVED** [review]
lgtm

File: src/call_log/call_log.c [hunk 1]
```diff
@@ -217,4 +217,34 @@ __wt_call_log_open_session(WT_SESSION_IMPL *session, int ret_val)
...
+    conn = S2C(session);
+
+    WT_RET(__call_log_print_start(session, "connection", "query_timestamp"));
+
+    /* Input arguments for the API call query_timestamp. */
+    WT_RET(__wt_snprintf(config_buf, sizeof(config_buf), "\"config\": \"%s\"", config));
+    WT_RET(__call_log_print_input(session, 1, config_buf));
+
+    /* Output arguments for the API call query_timestamp. */
+    WT_RET(__wt_snprintf(objectid_buf, sizeof(objectid_buf), "\"object_id\": \"%p\"", conn));
```
Comment (Sean04) [inline src/call_log/call_log.c:240]: You can change this now to reflect WT-9215

Comment (Sean04) [reply src/call_log/call_log.c:240]: That is to move the object id outside of the output array.

Comment (Siddhartha8899) [reply src/call_log/call_log.c:240]: Yup on it now.

Comment (Siddhartha8899) [reply src/call_log/call_log.c]: Updated in [081d141](https://github.com/wiredtiger/wiredtiger/pull/8095/commits/081d141a9fc378c9cf68b8a059012692ce84bcbe)

**Review by Sean04 — APPROVED** [review]
LGTM

**Pull request merged**